cmake/modules/CMakeLists.txt

```diff
@@ -96,6 +96,7 @@ set(IMEX_CONFIG_INCLUDE_EXPORTS "include(\"\${IMEX_CMAKE_DIR}/IMEXTargets.cmake\
 set(IMEX_CONFIG_INCLUDE_DIRS
     "${IMEX_SOURCE_DIR}/include"
     "${IMEX_BINARY_DIR}/include"
+    "${IMEX_BINARY_DIR}/tools/Imex/include"
```

Comment:
This path is only valid when building IMEX as an LLVM external project and setting Imex as the external project name.
I mean, using option 1 in README.md and passing something like
   -DLLVM_EXTERNAL_PROJECTS="Imex" \
   -DLLVM_EXTERNAL_IMEX_SOURCE_DIR=../mlir-extensions

Reply:
You would need to change to level CMakeLists.txt
https://github.com/intel/mlir-extensions/blob/main/CMakeLists.txt#L268
To something like.
set(IMEX_GENERATED_HEADER_DIR ${CMAKE_CURRENT_BINARY_DIR})
include_directories(${IMEX_GENERATED_HEADER_DIR}/include)
And then change
https://github.com/intel/mlir-extensions/blob/main/cmake/modules/CMakeLists.txt#L98
to
"${IMEX_GENERATED_HEADER_DIR}/include"

Reply:
@AllanZyne Thanks for catching this bug.

Reply:
Thank you for your suggestions, @silee2. Done.